### PR TITLE
Components: Tests for the PostRelativeTime component

### LINKED ISSUE #7943: Components: Add tests for <PostRelativeTime /> component
The [`<PostRelativeTime />` component](https://github.com/Automattic/wp-calypso/tree/master/client/blocks/post-relative-time) (block) is currently not tested. We should seek to add tests to verify expected render behavior of the component. [Enzyme](http://airbnb.io/enzyme/) (and [`chai-enzyme` assertions](https://github.com/producthunt/chai-enzyme)) may be useful utilities for rendering and asserting expected render results. Ideas for test cases may include some or all of the following:

- Render returns expected element
 - Optionally rendered components are or are not present
 - Modifier classes by incoming props or component state
- User interactions have expected results
- Impact of lifecycle methods when props or state change during lifetime of component

When submitting a pull request, reference this issue.

### PR DESCRIPTION
**Pull request by omarjackman**
This pull request adds some tests for the PostRelativeTime component

## How to test

Run `npm run test-client client/blocks/post-relative-time/`

fixes #7943

### CODE REVIEW

File: client/blocks/post-relative-time/test/index.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect, assert } from 'chai';
```
Comment (gziolo) [inline client/blocks/post-relative-time/test/index.jsx]: I would recommend using only one assertion style in a single file, in this case `expect` or `assert`.

Comment (omarjackman) [inline client/blocks/post-relative-time/test/index.jsx]: done in c1986b7

**Comment by tyxla** [comment]
@aduth thanks, good point - done 👍

**Comment by tyxla** [comment]
Rebased and LGTM 🚢

**Pull request merged**